fixtures/insights.json

```diff
@@ -136,7 +136,6 @@
           "autonomous_system_organization": "Linkem IR WiMax Network",
           "connection_type": "Cable/DSL",
           "domain": "example.com",
-          "ip_address": "1.2.3.4",
```

Comment:
Why was this removed?

Reply:
The "Add json linter" commit caught the duplicate `ip_address` field in this object. I kept the second instance of this field since that should be the one ultimately getting used after the first is overridden.

Reply:
Ahh, that makes sense. I should have looked more closely.